Mailchimp tags dest

mohan-h opened the pull request:
Initial commit

mohan-h committed:
Mailchimp Tags destination

Initial commit

mohan-h committed:
Added errors section

mohan-h committed:
Minor edit

vfalon commented on help/destinations/catalog/email-marketing/mailchimp-tags.md:
```diff
@@ -0,0 +1,173 @@
+---
+title: Mailchimp Tags
```
Is Tags supposed to be capitalized? Looking at their website (https://mailchimp.com/help/getting-started-tags/#About_tags), it seems it's all lowercase. Let's clarify if it should be `Mailchimp Tags` or `Mailchimp tags` both in the docs and in the UI.

mohan-h replied:
It is `T`ags.

vfalon commented on help/destinations/catalog/email-marketing/mailchimp-tags.md:
```diff
@@ -0,0 +1,173 @@
+
+[!DNL Mailchimp Tags] uses [audiences](https://mailchimp.com/help/getting-started-audience/) and [tags](https://mailchimp.com/help/getting-started-tags/) to manage your contact information. Tags are labels using which you can organize your contacts and label them for your internal categorization within [!DNL Mailchimp].
+
+This [!DNL Adobe Experience Platform] [destination](/help/destinations/home.md) leverages the [[!DNL Mailchimp Batch subscribe or unsubscribe API]](https://mailchimp.com/developer/marketing/api/lists/batch-subscribe-or-unsubscribe/) endpoint. You can add contacts or update tags of existing [!DNL Mailchimp] contacts within an existing [!DNL Mailchimp] audience after activating them within a new se ...
```
Lower case "batch subscribe"

vfalon commented on help/destinations/catalog/email-marketing/mailchimp-tags.md:
```diff
@@ -0,0 +1,173 @@
+
+### Send emails to contacts for marketing campaigns {#use-case-send-emails}
+
+The sales department of an organization wants to broadcast an email based marketing campaign to a curated list of contacts. The organizations marketing team can add new contacts or update tags of existing [!DNL Mailchimp] contacts, within an existing [!DNL Mailchimp] audience through Experience Platform, build segments from their own offline data, and send these segments to [!DNL Mailchimp Tags]. ...
```
organization's

vfalon replied:
Let's break up the second phrase into multiple sentences. It's rather difficult to follow what you mean there.

vfalon commented on help/destinations/catalog/email-marketing/mailchimp-tags.md:
```diff
@@ -0,0 +1,173 @@
+
+Before activating data to the [!DNL Mailchimp Tags] destination, you must have a [schema](/help/xdm/schema/composition.md), a [dataset](https://experienceleague.adobe.com/docs/platform-learn/tutorials/data-ingestion/create-datasets-and-ingest-data.html?lang=en), and [segments](https://experienceleague.adobe.com/docs/platform-learn/tutorials/segments/create-segments.html?lang=en) created in [!DNL  ...
+
+### Prerequisites in [!DNL Mailchimp Tags] {#prerequisites-destination}
```
I think there is a terminology confusion here. I don't think `Mailchimp Tags` is an actual name of a product, but please double check this with the team. Instead, my understanding is that tags are a feature of the Mailchimp platform.

So this heading would be ### Prerequisites in [!DNL Mailchimp], since customers need to log in to their Mailchimp account, where they can configure their tags.

Again, please double check if this is correct.

mohan-h replied:
Correct `Mailchimp Tags` is not a product however here we are referring to prerequisites for the destination. Have added the word destination to make it clearer and read as `### Prerequisites for the [!DNL Mailchimp Tags] destination {#prerequisites-destination}`. Note the subsequent sub-heading `#### You need to have a [!DNL Mailchimp] account {#prerequisites-account}` refers to `Mailchimp`.

vfalon commented on help/destinations/catalog/email-marketing/mailchimp-tags.md:
```diff
@@ -0,0 +1,173 @@
+
+#### Identify [!DNL Mailchimp] data center {#identify-data-center}
+
+Next you need to identify your Mailchimp domain, to do this log in to your [!DNL Mailchimp] account and check the URL in your browser.
```
Let's break this phrase up. 

Next you need to identify your Mailchimp domain. To do this, log in to your [!DNL Mailchimp] account and check the URL in your browser.

vfalon commented on help/destinations/catalog/email-marketing/mailchimp-tags.md:
```diff
@@ -0,0 +1,173 @@
+#### Identify [!DNL Mailchimp] data center {#identify-data-center}
+
+Next you need to identify your Mailchimp domain, to do this log in to your [!DNL Mailchimp] account and check the URL in your browser.
+You’ll notice a value similar to *`us19`.admin.mailchimp.com*, the `us19` prefix is the value you will require.
```
You’ll notice a value similar to *`us19`.admin.mailchimp.com*. Write down the prefix (`us19` in this example). You will need this prefix when connecting to the destination.

vfalon commented on help/destinations/catalog/email-marketing/mailchimp-tags.md:
```diff
@@ -0,0 +1,173 @@
+
+### Guardrails {#guardrails}
+
+Refer to the [!DNL Mailchimp] [rate limits](https://mailchimp.com/developer/marketing/docs/fundamentals/#api-limits) which details the limits imposed by the [!DNL Mailchimp] API.
```
Refer to the [!DNL Mailchimp] [rate limits](https://mailchimp.com/developer/marketing/docs/fundamentals/#api-limits) for detailed information about the limits imposed by the [!DNL Mailchimp] API.

vfalon commented on help/destinations/catalog/email-marketing/mailchimp-tags.md:
```diff
@@ -0,0 +1,173 @@
+
+| Target Identity | Description | Considerations |
+|---|---|---|
+| Email | Contact Email Address | Mandatory |
```
Contact email address

vfalon commented on help/destinations/catalog/email-marketing/mailchimp-tags.md:
```diff
@@ -0,0 +1,173 @@
+
+### Authenticate to destination {#authenticate}
+
+To authenticate to the destination, select **[!UICONTROL Connect to destination]**. You will be navigated to the [!DNL Mailchimp Log in and authorize] page. Enter your [!DNL Mailchimp] account credentials and select [!DNL Log In].
```
You will be redirected to the [!DNL Mailchimp] authorization page.

vfalon replied:
"Enter your [!DNL Mailchimp] account credentials and select [!DNL Log In]."

I believe there is a screenshot missing here. The image below shows the Connect to destination part, not the Mailchimp log in part.

vfalon commented on help/destinations/catalog/email-marketing/mailchimp-tags.md:
```diff
@@ -0,0 +1,173 @@
+
+![Mailchimp App screenshot confirmation popup to give permissions to the Experience Platform app access to Mailchimp.](../../assets/catalog/email-marketing/mailchimp-tags/allow-app.png)
+
+If the details provided are valid, the UI displays a message: *You successfully connected to Mailchimp Tags account* message and a **[!UICONTROL Connected]** status with a green check mark, you can then proceed to the next step.
```
If the details provided are valid, the Platform UI displays the following message along with a **[!UICONTROL Connected]** status with a green check mark: *[!DNL You successfully connected to Mailchimp Tags account]*. You can then proceed to the next step.

vfalon commented on help/destinations/catalog/email-marketing/mailchimp-tags.md:
```diff
@@ -0,0 +1,173 @@
+| **[!UICONTROL Name]** | A name by which you will recognize this destination in the future. |
+| **[!UICONTROL Description]** | A description that will help you identify this destination in the future. |
+| **[!UICONTROL Data Center]** | Your [!DNL Mailchimp] account `Data Center`. Refer to the [Identify [!DNL Mailchimp] data center](#identify-data-center) section for any guidance. |
+| **[!UICONTROL Audience List ID]** | After you provide the value of your `Data Center` entries in this select dropdown are automatically populated with the audience names from your account. You need to select the audience of interest to which you will add or update the contact details from selected Platform segments. |
```
After you enter your [!UICONTROL Data Center] prefix, this dropdown is automatically populated with the audience names from your account. Select the audience that you want to update with data from Platform.

vfalon commented on help/destinations/catalog/email-marketing/mailchimp-tags.md:
```diff
@@ -0,0 +1,173 @@
+To correctly map your XDM fields to the [!DNL Mailchimp Tags] destination fields, follow the steps below:
+
+1. In the **[!UICONTROL Mapping]** step, select **[!UICONTROL Add new mapping]**. You will see a new mapping row on the screen.
+1. In the **[!UICONTROL Select source field]** window, choose the **[!UICONTROL Select identity namespace]** and select `Email`.
```
In the **[!UICONTROL Select source field]** screen, choose  **[!UICONTROL Select identity namespace]** and select the `Email` identity namespace.

Also, let's add a screenshot for this step.

vfalon commented on help/destinations/catalog/email-marketing/mailchimp-tags.md:
```diff
@@ -0,0 +1,173 @@
+
+1. In the **[!UICONTROL Mapping]** step, select **[!UICONTROL Add new mapping]**. You will see a new mapping row on the screen.
+1. In the **[!UICONTROL Select source field]** window, choose the **[!UICONTROL Select identity namespace]** and select `Email`.
+1. In the **[!UICONTROL Select target field]** window, choose the **[!UICONTROL Select identity namespace]** and select `Email`.
```
In the **[!UICONTROL Select target field]** screen, choose **[!UICONTROL Select identity namespace]** and select the `Email` identity namespace.

Also, let's add a screenshot for this step.

vfalon commented on help/destinations/catalog/email-marketing/mailchimp-tags.md:
```diff
@@ -0,0 +1,173 @@
+
+To validate that you have correctly set up the destination, follow the steps below:
+
+1. Log in to the [[!DNL Mailchimp]](https://login.mailchimp.com/) website. Then navigate to the **[!DNL Audience]** > **[!DNL All Contacts]** page and check if the contacts from the segment have been added and contacts within the segment have been updated with the segment name.
```
Log in to your [[!DNL Mailchimp]](https://login.mailchimp.com/) account.

vfalon commented on help/destinations/catalog/email-marketing/mailchimp-tags.md:
```diff
@@ -0,0 +1,173 @@
+
+## Errors and troubleshooting {#errors-and-troubleshooting}
+
+Refer to the [[!DNL Mailchimp] Errors](https://mailchimp.com/developer/marketing/docs/errors/) page for a comprehensive list of status and error codes with explanations.
```
Refer to the [!DNL Mailchimp] [errors page](https://mailchimp.com/developer/marketing/docs/errors/)

vfalon reviewed (changes requested):
Hey @mohan-h ! Thanks for submitting the Mailchimp documentation. It looks great, just a few comments to address before we get it ready.

mohan-h committed:
Updates against review

mohan-h committed:
Merge branch 'AdobeDocs:main' into mailchimp-tags-dest

mohan-h committed:
Authorization image updated

New images includes the Adobe logo

mohan-h committed:
DC field change to select

mohan-h committed:
Merge remote-tracking branch 'upstream/main' into mailchimp-tags-dest

mohan-h committed:
Authentication changed

mohan-h committed:
Added note for mailchimp interest categories

mohan-h committed:
Corrected label

vfalon commented on help/destinations/catalog/email-marketing/mailchimp-tags.md:
```diff
@@ -0,0 +1,199 @@
+
+### Send emails to contacts for marketing campaigns {#use-case-send-emails}
+
+The sales department of an organization wants to broadcast an email based marketing campaign to a curated list of contacts. The lists of contact are received in batches from different offline sources and therefore need to be tracked. The team identifies an existing [!DNL Mailchimp] audience and starts building the Experience Platform segment's into which the contacts from each list are added. Aft ...
```
Line 21: "starts building the Experience Platform segment's "  ---> "starts building the Experience Platform audiences"

vfalon replied:
"The lists of contact are received" ---> "The lists of contacts is received"

vfalon replied:
"After sending these segments to [!DNL Mailchimp Tags] if any contacts do not exist in the selected [!DNL Mailchimp] audience they get added with an associated tag having the segment name the contact belongs to."

Please rephrase to:

"After sending these audiences to [!DNL Mailchimp Tags], if any contacts do not exist in the selected [!DNL Mailchimp] audience, they get added with an associated tag which includes the audience name the contact belongs to."

vfalon commented on help/destinations/catalog/email-marketing/mailchimp-tags.md:
```diff
@@ -0,0 +1,199 @@
+
+You need your [!DNL Mailchimp] **API key** to authenticate the [!DNL Mailchimp Interest Categories] destination against your [!DNL Mailchimp] account. The **API key** serves as the **Password** when you [authenticate the destination](#authenticate).
+
+If you do not have your **API key**, Sign in to your account and refer to the [[!DNL Mailchimp] Generate your API key](https://mailchimp.com/developer/marketing/guides/quick-start/#generate-your-api-key) documentation to create one. 
```
Line 43: lowercase s in "Sign in"

vfalon replied:
Please reword this to:

If you do not have your **API key**, sign in to your [!DNL Mailchimp] account and refer to the [!DNL Mailchimp] documentation on [how to generate your API key](https://mailchimp.com/developer/marketing/guides/quick-start/#generate-your-api-key).

vfalon commented on help/destinations/catalog/email-marketing/mailchimp-tags.md:
```diff
@@ -0,0 +1,199 @@
+>
+>If you generate the **API key**, write it down as you will not be able to access it after generation.
+
+#### Identify [!DNL Mailchimp] data center {#identify-data-center}
```
Line 51: Identify your [!DNL Mailchimp] data center {#identify-data-center}

vfalon commented on help/destinations/catalog/email-marketing/mailchimp-tags.md:
```diff
@@ -0,0 +1,199 @@
+
+Next, you must identify your [!DNL Mailchimp] data center. To do this, log in to your [!DNL Mailchimp] account and navigate to the **API keys section** of your account. 
+
+The value is the first part of the URL you see in your browser. If the URL is *https://`us14`.mailchimp.com/account/api/*, then the data center is `us14`. 
```
Line 55: The data center ID is the first section of the URL you see in your browser. If the URL is *https://`us14`.mailchimp.com/account/api/*, then the data center ID is `us14`.

vfalon commented on help/destinations/catalog/email-marketing/mailchimp-tags.md:
```diff
@@ -0,0 +1,199 @@
+
+The value is the first part of the URL you see in your browser. If the URL is *https://`us14`.mailchimp.com/account/api/*, then the data center is `us14`. 
+
+It's also appended to your API key in the form *key-dc*; if your API key is `0123456789abcdef0123456789abcde-us14`, then the data center is `us14`. 
```
Line 57: The data center ID is also appended to your API key in the form *key-dc*. For example, if your API key is `0123456789abcdef0123456789abcde-us14`, then the data center is `us14`.

vfalon commented on help/destinations/catalog/email-marketing/mailchimp-tags.md:
```diff
@@ -0,0 +1,199 @@
+
+It's also appended to your API key in the form *key-dc*; if your API key is `0123456789abcdef0123456789abcde-us14`, then the data center is `us14`. 
+
+Write down the data center value *(`us14` in this example)*, you need this value when you [fill in destination details](#destination-details).
```
Line 59: Write down the data center value *(`us14` in this example)*. You will need this value when you [fill in the destination details](#destination-details).

vfalon commented on help/destinations/catalog/email-marketing/mailchimp-tags.md:
```diff
@@ -0,0 +1,199 @@
+
+| Target Identity | Description | Considerations |
+|---|---|---|
+| Email | Contact email Address | Mandatory |
```
Line 73: Update the description to: The email address of the contact.

vfalon commented on help/destinations/catalog/email-marketing/mailchimp-tags.md:
```diff
@@ -0,0 +1,199 @@
+
+This section describes which type of audiences you can export to this destination.
+
+| Audience origin | Supported | Description | 
```
Please update the supported audiences table to:


| Audience origin | Supported | Description | 
---------|----------|----------|
| [!DNL Segmentation Service] | ✓ | Audiences generated through the Experience Platform [Segmentation Service](../../../segmentation/home.md).|
| Custom uploads | X | Audiences [imported](../../../segmentation/ui/overview.md#import-audience) into Experience Platform from CSV files. |

vfalon commented on help/destinations/catalog/email-marketing/mailchimp-tags.md:
```diff
@@ -0,0 +1,199 @@
+
+| Item | Type | Notes |
+---------|----------|---------|
+| Export type | **[!UICONTROL Profile-based]** | <ul><li>You are exporting all members of a segment, together with the desired schema fields *(for example: email address, phone number, last name)*, according to your field mapping.</li><li> For each selected segment in Platform, the corresponding [!DNL Mailchimp Tags] segment status gets updated with its segment status from Platform.</li></ul>|
```
Line 93: "You are exporting all members of a segment, " --> "You are exporting all members of an audience, "

vfalon replied:
"For each selected segment in Platform, the corresponding [!DNL Mailchimp Tags] segment status gets updated with its segment status from Platform." --> "For each audience selected in Platform, the corresponding [!DNL Mailchimp Tags] segment status gets updated with the audience status from Platform."

vfalon commented on help/destinations/catalog/email-marketing/mailchimp-tags.md:
```diff
@@ -0,0 +1,199 @@
+| Item | Type | Notes |
+---------|----------|---------|
+| Export type | **[!UICONTROL Profile-based]** | <ul><li>You are exporting all members of a segment, together with the desired schema fields *(for example: email address, phone number, last name)*, according to your field mapping.</li><li> For each selected segment in Platform, the corresponding [!DNL Mailchimp Tags] segment status gets updated with its segment status from Platform.</li></ul>|
+| Export frequency | **[!UICONTROL Streaming]** | Streaming destinations are "always on" API-based connections. As soon as a profile is updated in Experience Platform based on segment evaluation, the connector sends the update downstream to the destination platform. Read more about [streaming destinations](/help/destinations/destination-types.md#streaming-destinations).|
```
"As soon as a profile is updated in Experience Platform based on segment evaluation,"  --> "As soon as a profile is updated in Experience Platform based on audience evaluation,"

vfalon commented on help/destinations/catalog/email-marketing/mailchimp-tags.md:
```diff
@@ -0,0 +1,199 @@
+
+| Field | Description |
+| --- | --- |
+| **[!UICONTROL Username]** | Your [!DNL Mailchimp Interest Categories] username. |
```
Your [!DNL Mailchimp Interest Categories] username.

I believe this should say Mailchimp Tags.

vfalon reviewed (changes requested):
@mohan-h I've reviewed the latest changes and left some more comments to address. Please let me know if you have any questions. Thank you!

mohan-h committed:
Fixes for review guidance.

mohan-h committed:
We now support custom uploads

We now support custom uploads, see the [supported audiences](#supported-audiences) section.
